client/src/components/CreateFriendsList/User.js

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components';
+import { useState, useEffect } from 'react';
+
+import ListItem from '@material-ui/core/ListItem';
+import Avatar from '@material-ui/core/Avatar';
+import Divider from '@material-ui/core/Divider';
+import Button from '@material-ui/core/Button';
+
+export default function Suggestions ({ user, newFriendsList, setNewFriendsList }) {
+    const [added, setAdded] = useState(false);
+    
+    useEffect(() => {
+        const checkIfAdded = () => {
+            return (
+                newFriendsList.friends.length > 0 && 
+                    newFriendsList.friends.some(e => e._id === user._id) 
+                            ? setAdded(true)
+                            : setAdded(false)
```

Comment:
```suggestion
setAdded(newFriendsList.friends.some(e => e._id === user._id) )
```

to reduce lines of code